### DISCUSSION TITLE
Compatibility with grml-zsh-config

### DISCUSSION THREAD

**Discussion by chrisr87** [marlonrichert/zsh-autocomplete#504] [Ideas]
I installed grml-zsh-config with all defaults and zsh-autocomplete with all defaults and when I press `tab`, a list of directories (or commands, depending on what it was trying to complete) flash and then disappears. Anyone have an idea on how to fix this or is zsh-autocomplete just not compatible with grml-zsh-config out of the box?

**Comment by marlonrichert** [comment]
It’s most definitely possible, but it will require some investigation to figure out how. I made an issue for it: #505

**Comment by chrisr87** [comment]
I've been fiddling and skimming through the configs and I changed

```diff
-zstyle ':autocomplete:*' widget-style complete-word
+zstyle ':autocomplete:*' widget-style menu-select
```

and it works now. Not sure why though. For reference, this is the base `zsh-autocomplete` config I added:
https://github.com/marlonrichert/zsh-autocomplete/blob/main/.zshrc

edit: nvm, while the change I made did stop it from flashing and disappearing, most of the keybindings don't work as expected
